src/bci_build/package.py

```diff
@@ -1463,7 +1468,9 @@ def _get_python_kwargs(
         symlink_py_and_pydoc = rf"""ln -s /usr/bin/python{py3_ver} /usr/local/bin/python3; \
     ln -s /usr/bin/pydoc{py3_ver} /usr/local/bin/pydoc"""
 
-        kwargs["config_sh_script"] = symlink_py_and_pydoc
+        kwargs["config_sh_script"] = (
+            "install -d -m 0755 /root/.local/bin; "
```

Comment:
Won't pipx do this for you?

Reply:
pipx does create the directory for you once you install something that requires it to do so. however it doesn't add it to the $PATH, which is the real change here. and I don't want to add things to the $PATH that haven't been created in the container with proper permissions, I find that unclean.